src/gf.c

```diff
@@ -1296,10 +1300,10 @@ static jl_value_t *check_ambiguous_matches(jl_typemap_t *defs, jl_typemap_entry_
         else
             va = NULL;
     }
-    struct ambiguous_matches_env env;
-    env.match.fptr = fptr;
-    env.match.type = (jl_value_t*)type;
-    env.match.va = va;
+    struct ambiguous_matches_env env = {.match = {.fptr = fptr, .type = (jl_value_t*)type, .va = va}};
+    //env.match.fptr = fptr;
+    //env.match.type = (jl_value_t*)type;
+    //env.match.va = va;
```

Comment:
delete?

Reply:
Ah, yes, I should do that cleanup. How do we feel about starting to use the C99 initialization syntax? (this it not valid C++ syntax, though something similar will be added in C++20)